Modules/Core/Common/src/itkConfigure.h.in

```diff
@@ -170,12 +170,21 @@
 #define ITK_USE_POOL_MULTI_THREADER 1
 #endif
 
-#if defined(ITK_USE_TBB)
-#define ITK_DEFAULT_THREADER TBB
-#elif defined(ITK_USE_POOL_MULTI_THREADER)
-#define ITK_DEFAULT_THREADER Pool
-#else
-#define ITK_DEFAULT_THREADER Platform
+
+#define ITK_DEFAULT_THREADER @ITK_DEFAULT_THREADER@
+#define ITK_DEFAULT_THREADER_@ITK_DEFAULT_THREADER@
```

Comment:
This line is almost the same as line 174. The change from space to underscore is hard to spot :)

Reply:
Adding comment.